README.md

```diff
@@ -12,7 +12,42 @@ The aim of govuk-docker is to make it easy to develop any GOV.UK app. It achieve
 
 ## Installation
 
-[Instructions for how to install and setup govuk-docker are here.](docs/installation.md)
+**First make sure you have the following dependencies.**
+
+- [brew](https://brew.sh/)
+- [git](https://git-scm.com)
+- A directory `~/govuk` in your home directory
+
+**Next, add the following line to your bash/zsh config.**
+
+```
+export PATH=$PATH:~/govuk/govuk-docker/exe
+```
```

Comment:
You may want a step telling them to run `source ~/.zshrc` or `source ~/.bashrc` so the path is updated for the session.

Reply:
Good idea. I've added some pointers about that.